include/modem/pdn.h

```diff
@@ -60,6 +60,7 @@ enum pdn_event {
 	PDN_EVENT_DEACTIVATED,	/**< PDN connection deactivated */
 	PDN_EVENT_IPV6_UP,	/**< PDN has IPv6 connectivity */
 	PDN_EVENT_IPV6_DOWN,	/**< PDN has lost IPv6 connectivity */
+	PDN_EVENT_DETACHED,	/**< PDN connection detached */
```

Comment:
If we keep this, I would add a bit more information, for example `/**< All PDN connections deactivated */`

Reply:
This is not really a PDN event, it's a full network detach. Maybe it should be named `PDN_EVENT_NETWORK_DETACH` to clearly show this?

Reply:
Yeah, I think that's clearer.